### PR TITLE
Samples/update audio

### PR DESCRIPTION
**Pull request by shivamMg**
https://github.com/fossasia/open-event-orga-server/issues/1454

@aviaryan Please review.

### CODE REVIEW

File: samples/ots16/event.json [hunk 1]
```diff
@@ -20,38 +20,6 @@
     "organizer_name": "FOSSASIA",
     "privacy": "public",
     "schedule_published_on": "2016-05-04 09:00:00",
-    "social_links": [
-        {
```
Comment (aviaryan) [inline samples/ots16/event.json]: Why did you remove social_links ? Import JSON files should reflect the APIs and event GET API has social_links so I think we should keep it. 
Also the export of event will have social_links field in event.json (as they are based on APIs ) .

Comment (shivamMg) [inline samples/ots16/event.json]: They seemed redundant as Social links were already in social_links.json, where they are actually created from. OK, I'll reset commit.
